### PR TITLE
Replace Presto with Trino in CBO blog

### PR DESCRIPTION
**Pull request by colebow**
Behold, the Trinoification. No URLs/links updated to not break things.

### CODE REVIEW

**Comment by bitsondatadev** [comment]
Curious if we think it's worth updating the filenames which include presto as well. It likely won't be an issue of SEO but figured I would ask the question.

Also, we should coordinate with @monimiller to update the Starburst cross-post of this: https://www.starburst.io/blog/introduction-to-presto-cost-based-optimizer/

Monica, would you mind updating this on the Starburst side and adding @findepi as the author please?

File: _posts/2019-07-04-cbo-introduction.md [hunk 1]
```diff
@@ -258,21 +258,18 @@ filter condition or are returned from the join.
...
-Enabling Presto CBO is really simple:
+Enabling Trino CBO is really simple:
 
 - set `optimizer.join-reordering-strategy=AUTOMATIC` and
   `join-distribution-type=AUTOMATIC` in your `config.properties`,
 - [analyze]({{site.url}}/docs/current/sql/analyze.html) your tables,
 - no, there is no third step. That’s it!
 
-Take Presto CBO for a spin today and let us know what is *your* Presto
-experience!
```
Comment (findepi) [inline _posts/2019-07-04-cbo-introduction.md:276]: why removed?

Comment (mosabua) [reply _posts/2019-07-04-cbo-introduction.md:276]: I agree.. why remove and not just also rename

File: _posts/2019-07-04-cbo-introduction.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Blog](https://www.starburstdata.com/technical-blog/introduction-to-presto-cost-b
```
Comment (monimiller) [inline _posts/2019-07-04-cbo-introduction.md:18]: Do we want to edit the Starburst permalink? If so, need this updated as well

**Comment by mosabua** [comment]
Also .. we should NOT rename the file since that would break any incoming link unless we set up a redirect .. and that would hurt SEO .. so lets leave the filename

File: _posts/2019-07-04-cbo-introduction.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
 ---
 layout: post
-title:  "Introduction to Presto Cost-Based Optimizer"
+title:  "Introduction to Trino Cost-Based Optimizer"
 author: Piotr Findeisen, Starburst Data
 excerpt_separator: <!--more-->
 ---
 
-The Cost-Based Optimizer (CBO) in Presto achieves stunning results in industry
+Edited 15 June 2022: Updated this post to use the Trino project name.
```
Comment (mosabua) [inline _posts/2019-07-04-cbo-introduction.md:8]:
```suggestion
Last edited 15 June 2022: Update to use the Trino project name.
```

**Review by mosabua — APPROVED** [review]

**Review by mosabua — APPROVED** [review]
Looks good now.

**Review by findepi — APPROVED** [review]

**Comment by findepi** [comment]
Preview https://deploy-preview-305--trino.netlify.app/blog/2019/07/04/cbo-introduction.html lgtm

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**